src/System.Reflection/tests/TypeDerivedTests.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    internal class MockDerivedType : BaseMockType
+    {
+        public override Type UnderlyingSystemType => null;
+        protected override TypeAttributes GetAttributeFlagsImpl() => new TypeAttributes();
```

Comment:
It may be better to use `TypeAttributes.Class` instead of `new TypeAttributes()`.